config/openiotsdk/cmake/sdk.cmake

```diff
@@ -65,28 +67,48 @@ FetchContent_Declare(
 # Open IoT SDK configuration
 set(IOTSDK_FETCH_LIST
     mcu-driver-reference-platforms-for-arm
+    mbed-critical
     cmsis-5
     cmsis-freertos
     mbedtls
     lwip
-    cmsis-sockets-api
     trusted-firmware-m
 )
 
 set(MDH_PLATFORM ARM_AN552_MPS3)
 set(VARIANT "FVP")
 set(FETCHCONTENT_QUIET OFF)
-set(TFM_PLATFORM ${OPEN_IOT_SDK_EXAMPLE_COMMON}/tf-m/targets/an552)
-set(TFM_PSA_FIRMWARE_UPDATE ON)
-set(MCUBOOT_IMAGE_VERSION_NS ${TFM_NS_APP_VERSION})
-set(TFM_CMAKE_ARGS "-DCONFIG_TFM_ENABLE_FP=ON;-DTFM_PROFILE=profile_medium;-DTFM_EXCEPTION_INFO_DUMP=ON;-DCONFIG_TFM_HALT_ON_CORE_PANIC=ON;-DTFM_ISOLATION_LEVEL=1;-DTFM_MBEDCRYPTO_PLATFORM_EXTRA_CONFIG_PATH=${OPEN_IOT_SDK_CONFIG}/mbedtls/mbedtls_config_psa.h;-DMBEDCRYPTO_BUILD_TYPE=${CMAKE_BUILD_TYPE};-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}")
+set(TFM_CMAKE_ARGS
```

Comment:
Why are we not passing `MBEDCRYPTO_BUILD_TYPE` and `CMAKE_BUILD_TYPE` into TF-M CMake args ?